pyspecProcScripts/phasing.py

```diff
@@ -242,16 +238,20 @@ def hermitian_function_test(
     orig_dt = s.get_ft_prop(direct, "dt")
     s.ft(direct)
     s = s[direct:frq_range]
-    s.ift(direct, pad=2048 * 8)
+    s.ift(direct, pad=1024 * 16)
+    new_dt = s.get_ft_prop(direct, "dt")
+    non_aliased_range = r_[3,-3]*int(orig_dt/new_dt)
+    ini_delay = non_aliased_range[0]*new_dt
+    s = s[direct,non_aliased_range[0]:non_aliased_range[1]]
     if fl is not None:
-        plt.rcParams["figure.figsize"] = [7.5, 3.5]
-        plt.rcParams["figure.autolayout"] = True
         fig, ax_list = subplots(2, 3, figsize=(15, 15))
         fl.next("Hermitian Function Test Diagnostics", fig=fig)
-        s = s[direct:(ini_delay, None)]
```

Comment:
this was done inside the `if fl is not None` -- which is a big no-no b/c data would be processed different based on whether you called the diagnostic plots or not.